README.md

```diff
@@ -1,73 +1,40 @@
-# electron-vite-react
+# BitGo Wallet Recovery Wizard
 
-[![awesome-vite](https://awesome.re/mentioned-badge.svg)](https://github.com/vitejs/awesome-vite)
-![GitHub stars](https://img.shields.io/github/stars/caoxiemeihao/vite-react-electron?color=fa6470)
-![GitHub issues](https://img.shields.io/github/issues/caoxiemeihao/vite-react-electron?color=d8b22d)
-![GitHub license](https://img.shields.io/github/license/caoxiemeihao/vite-react-electron)
-[![Required Node.JS >= 14.18.0 || >=16.0.0](https://img.shields.io/static/v1?label=node&message=14.18.0%20||%20%3E=16.0.0&logo=node.js&color=3f893e)](https://nodejs.org/about/releases)
+This tool is built for assisting customers with recovering coins from BitGo wallets.
 
-## 👀 Overview
+The application currently offers the following recoveries:
 
-📦 Ready out of the box  
-🎯 Based on the official [template-react-ts](https://github.com/vitejs/vite/tree/main/packages/create-vite/template-react-ts), project structure will be familiar to you  
-🌱 Easily extendable and customizable  
-💪 Supports Node.js API in the renderer process  
-🔩 Supports C/C++ native addons  
-🐞 Debugger configuration included  
-🖥 Easy to implement multiple windows
+Withdraw Funds without BitGo: 
 
-## 🛫 Quick start
+  * Unsigned Sweep: Build an unsigned transaction using the user and backup public keys, independent from any BitGo services. 
+  * Non-BitGo Recoveries: Build a transaction using the recovery KeyCard, independent from any BitGo services.
+  * Coin Support:
+      |       | Production    | Test          |
+      |-------| ------------- | ------------- |
+      |  AVAXC  | Yes   | Yes  |
+      |  BTC  | Yes   | Yes  |
+      |  BCH  | Yes   | No  |
+      |  BTG  | Yes   | No  |
+      |  DASH | Yes  | No  |
+      |  ETH  | Yes  | Yes  |
+      |  EOS  | Yes  | Yes  |
+      |  ERC20  | Yes  | Yes  |
+      |  LTC  | Yes  | No  |
+      |  TRX  | Yes  | Yes  |
+      |  XLM  | Yes  | Yes  |
+      |  XRP  | Yes  | Yes  |
+      |  ZEC  | Yes  | No  |
 
-```sh
-npm create electron-vite
-```
+## Installing and Downloading
 
-![electron-vite-react.gif](https://github.com/electron-vite/electron-vite-react/blob/main/public/electron-vite-react.gif?raw=true)
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
this project was not bootstrapped with Create React App. please read over your changes when you copy/paste stuff please.

Reply:
https://github.com/electron-vite/electron-vite-react this is what we used.